### PR TITLE
Remove top because etop is bundled into the heavy observer

### PR DESCRIPTION
**Pull request by codeadict**

### CODE REVIEW

File: priv/base/runner [hunk 1]
```diff
@@ -72,33 +71,17 @@ This is the primary script for controlling the $SCRIPT node.
...
+        be reached. Exit \`$SCRIPT attach\` by pressing Ctrl-C twice.
 
     attach-direct
         Attaches to the console of a $SCRIPT node running in the background
         using a directly connected FIFO, giving access to the Erlang shell
-        and runtime messages. Prints "Node did not respond to ping!" when the
-        node cannot be reached.
-        Exit \`$SCRIPT attach-direct\` by pressing \"Ctrl-D\".
+        and runtime messages. Prints "Node is not running!" when the node
+        cannot be reached. Exit \`$SCRIPT attach-direct\` by pressing Ctrl-D.
```
Comment (larshesel) [inline priv/base/runner]: What is this change?

File: priv/base/runner [hunk 2]
```diff
@@ -303,11 +286,10 @@ case "$1" in
         # Make sure a node is running
         node_up_check
 
-        echo "Remote Shell: Use \"Ctrl-G q\" to quit."
-        echo "q() or init:stop() will terminate the $SCRIPT node."
+        echo "Remote Shell: Use \"Ctrl-C a\" to quit. q() or init:stop() will terminate the $SCRIPT node."
         shift
         NODE_NAME=${NAME_ARG#* }
-        exec $ERTS_PATH/erl -name c_$$_$NODE_NAME -hidden -remsh $NODE_NAME -boot start_clean $COOKIE_ARG $NET_TICKTIME_ARG
+        exec $ERTS_PATH/erl -name c_$$_$NODE_NAME -hidden -remsh $NODE_NAME $COOKIE_ARG $NET_TICKTIME_ARG
```
Comment (larshesel) [inline priv/base/runner]: And the two aboves here? Is this from a newer copy of the runner script?

**Comment by codeadict** [comment]
I think is my editor that just messed up the diff. I was scared too but it
looks fine, I just removed the top things

**Comment by larshesel** [comment]
Can you revert the other changes than the ones about removing the top cmd?

**Comment by codeadict** [comment]
Done @larshesel, somehow my editor messed up stuff

**Pull request merged**